tests/TestCase/Model/Table/QueuedTasksTableTest.php

```diff
@@ -1,48 +1,48 @@
 <?php
-/**
- * @author MGriesbach@gmail.com
- * @license http://www.opensource.org/licenses/mit-license.php The MIT License
- * @link http://github.com/MSeven/cakephp_queue
- */
```

Comment:
Oh, that was a copy/paste misstake because i've created the Test again with the "bake" command